src/test/acceptance/features/manual-address.feature

```diff
@@ -42,6 +42,10 @@ Feature: Address
     When I do not enter the 'town or city' of an address
     Then I am informed that I need to enter an address on the 'town or city' field
 
+  Scenario: Do not enter in the county
```

Comment:
I'd rather not make County a required field - we could default to a sensible value ('UK'? - do this in the claimant service, not here) if it's not provided.

Reply:
It's required for allpay, and will be printed on the letters. Maybe one for the UX team?

Reply:
What @YetAnotherMatt said!!

Reply:
I've spoken to the UX team, their preference is for it not to be there at all, and if it does have to be then for it to not be mandatory. If it's not present in the OS places response we won't be asking them to add it manually, so we'll have to handle an empty value anyway. Let's have the County field for now, not mandatory, and default to 'UK' in the claimant service if it's not provided. We can revisit whether the field is necessary once we've got OS places and postcodes.io integrated.

Reply:
Ok, will make optional

Reply:
Can we make sure that we have a test that makes sure an application is successful with no county then please?

Reply:
Field is now optional. Complete application test will come in next PR due to number of files it involves changing.